pyanaconda/ui/gui/spokes/keyboard.py

```diff
@@ -520,7 +520,7 @@ def on_up_clicked(self, button):
             return
 
         store.swap(cur, prev)
-        if flags.can_touch_runtime_system("reorder runtime X layouts", touch_live=True):
+        if conf.system.can_localize_live:
```

Comment:
This can be `can_configure_runtime_X_layouts` as well.

Reply:
Changed to `can_configure_keyboard`.